packages/stream-caip10-link/src/util.ts

```diff
@@ -0,0 +1,13 @@
+import { AccountId } from 'caip'
+
+export function normalizeAccountId(accountId: string | AccountId): AccountId {
```

Comment:
Now we have this method in three places. Maybe we can just put it in `@ceramicnetwork/common`?

Reply:
One of them is different from the others, but yes. Will do.